js/app.js

```diff
@@ -0,0 +1,104 @@
+function createPerson4Request(url, id, propertyName) {
+  let oReq = new XMLHttpRequest();
+  oReq.addEventListener('load', changeWhat);
+  oReq.open('GET', url);
+  oReq.send();
+
+  function changeWhat() {
```

Comment:
Look into more meaningful function name
https://hackernoon.com/the-art-of-naming-variables-52f44de00aad